metasrv/src/configs/config.rs

```diff
@@ -13,11 +13,25 @@
 // limitations under the License.
 
 use clap::Parser;
+use common_exception::ErrorCode;
+use common_meta_raft_store::config as raft_config;
 use common_meta_raft_store::config::RaftConfig;
 use lazy_static::lazy_static;
 use serde::Deserialize;
 use serde::Serialize;
 
+macro_rules! load_field_from_env {
+    ($struct:expr, $field:tt, $field_type: ty, $env:expr) => {
+        if let Some(env_var) = std::env::var_os($env) {
+            $struct.$field = env_var
+                .into_string()
+                .expect(format!("connot convert {} to string", $env).as_str())
+                .parse::<$field_type>()
+                .expect(format!("cannot convert {} to {}", $env, stringify!($field_type)).as_str());
+        }
+    };
+}
```

Comment:
What about 
```
    ($field:tt, $field_type: ty, $env:expr) => {
   // ... 
   $field = ...
```

It seems `$struct` is always used together with`$field` and can be reduced.